indexers/txindex/kv/kv.go

```diff
@@ -592,30 +594,41 @@ func (txi *TxIndex) pruneTxs(from, to uint64) (uint64, error) {
 
 	for h := from; h < to; h++ {
 
+		// first all events are pruned associated to the same height
+		prunedEvents, err := txi.pruneEvents(h, batch)
+		if err != nil {
+			continue
```

Comment:
should we log here at least something so will know we had problem pruning?

Reply:
added